exams/pearson/download_test.py

```diff
@@ -288,13 +300,13 @@ def setUpTestData(cls):
             ]
             cls.failure_profiles = ExamProfileFactory.create_batch(2)
 
-        cls.success_results = [VCDCResult(
+        cls.success_results = ([VCDCResult(
```

Comment:
can you drop the `VCDCResult` invocation to a line below like you're doing at line 310?